### ISSUE TITLE
idevicepair error -5 

### ISSUE THREAD

**Issue by Enekololo** [libimobiledevice/libimobiledevice#1633]
`ERROR: Device 00008110-001A30E63E89A01E returned unhandled error code -5`

i keep having the same error and could not find the solution pls help me

**Comment by bclouthier** [comment]
Yes, I had the same issue. I could see the device but I could not pair the device or initate a backup. What worked for me on Fedora 41 was running the following command in the terminal, `sudo update-crypto-policies --set FEDORA40`. I'm running Fedora 41. Apparently, the crypto policy breaks the iOS devices ability to mount and share files and so you regress to Fedora 40's crypto policy. More info here, [https://bugzilla.redhat.com/show_bug.cgi?id=2321958.](url)

**Comment by dennis-benzinger** [comment]
There's a less invasive workaround at:
https://github.com/libimobiledevice/libimobiledevice/issues/1606#issuecomment-2543116644

**Comment by kfurkanbek** [comment]
Even though I agree with the solution suggested by @dennis-benzinger here,
I just downgraded, transferred and upgraded it back.

1. sudo update-crypto-policies --set FEDORA40
2. Restart
3. sudo update-crypto-policies --set DEFAULT

**Comment by F-Depi** [comment]
I can confirm that as of today with iOS 16.7 and fedora 43 (after an inconclusive hour of troubleshooting)
```
sudo update-crypto-policies --set LEGACY
```
followed by a reboot of ```usbmuxd```:
```
sudo systemctl restart usbmuxd
```
Made the command ```devicepair pair``` work and I could do a backup of the phone.
I tried the less invasive workaround [#1606 (comment)](https://github.com/libimobiledevice/libimobiledevice/issues/1606#issuecomment-2543116644) suggested by @dennis-benzinger before but it did not work.
One can put the system back to default with
```
sudo update-crypto-policies --set DEFAULT
```
afterwards.
